src/components/transactions/Bridge/BridgeConfig.ts

```diff
@@ -0,0 +1,124 @@
+// Note taken from
+// https://github.com/smartcontractkit/smart-contract-examples/blob/main/ccip-offchain/javascript/src/config/router.js
+import { ChainId } from '@aave/contract-helpers';
+
+type Config = {
+  sourceChainId: ChainId;
+  router: string;
+  chainSelector: string;
+  destinations: {
+    destinationChainId: ChainId;
+    onRamp: string;
+  }[];
+};
+
+export enum MessageExecutionState {
+  UNTOUCHED = 0,
+  IN_PROGRESS,
+  SUCCESS,
+  FAILURE,
+}
+
+// TODO: make testnet and mainnet config
```

Comment:
yeah can't forget also for supported networks as well

Reply:
Yeah, I added a function in this file, `getSupportedSourceChains`, it just returns all of the chain ids that are in the config. So essentially if it's not in the lane config, it's not a supported network.